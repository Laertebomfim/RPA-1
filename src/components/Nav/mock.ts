import { linkPros } from './Nav';

export const Mock: linkPros = [
  {
    children: 'link-1',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-2',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-3',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-4',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-5',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-6',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-7',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-8',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-9',
    http: 'http://mock.com',
    newtab: '_self',
  },

  {
    children: 'link-10',
    http: 'http://mock.com',
    newtab: '_self',
  },
];
