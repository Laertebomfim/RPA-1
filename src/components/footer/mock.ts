export const mock = {
  background: true,
  text: `
    <p>
        <a href='http://www.test.com'>Pellentesque habitant morbi tristique senectus</a>
    </p>
   `,
};
