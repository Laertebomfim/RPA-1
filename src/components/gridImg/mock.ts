export const mock = {
  background: true,
  TitleText: 'Lorem ipsum dolor',
  descriptionP:
    'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Molestias libero laborum, repudiandae iure consectetur commodi, error soluta modi dignissimos alias.',
  gridImg: [
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
    {
      TitleAlt: 'text',
      srcImg:
        'https://s1.static.brasilescola.uol.com.br/be/conteudo/images/imagem-em-lente-convexa.jpg',
    },
  ],
};
