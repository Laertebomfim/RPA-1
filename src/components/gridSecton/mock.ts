export const mock = {
  background: true,
  TitleText: 'Lorem ipsum dolor',
  descriptionP:
    'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Molestias libero laborum, repudiandae iure consectetur commodi, error soluta modi dignissimos alias.',
  grid: [
    {
      Title: 'text',
      description:
        'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Molestias libero laborum, repudiandae iure consectetur commodi, error soluta modi dignissimos alias expedita, ex culpa nemo velit explicabo mollitia ipsam incidunt debitis!',
    },
    {
      Title: 'text',
      description:
        'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Molestias libero laborum, repudiandae iure consectetur commodi, error soluta modi dignissimos alias expedita, ex culpa nemo velit explicabo mollitia ipsam incidunt debitis!',
    },
    {
      Title: 'text',
      description:
        'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Molestias libero laborum, repudiandae iure consectetur commodi, error soluta modi dignissimos alias expedita, ex culpa nemo velit explicabo mollitia ipsam incidunt debitis!',
    },
  ],
};
