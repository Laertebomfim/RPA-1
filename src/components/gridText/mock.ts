export const mock = {
  TitleText: 'my title',
  text: ['hi '],
  background: true,
};
