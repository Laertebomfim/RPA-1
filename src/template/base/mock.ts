import GridText from '../../components/gridText';
import { mock as informationMockGrid } from '../../components/gridText/mock';
import { BaseProps } from '.';

export const mock: BaseProps = {
  children: GridText(informationMockGrid),
  textFooter: `<p><a href='http://www.test.com'>Pellentesque habitant morbi tristique senectus</a>
</p>`,
  information: [
    { children: 'link', http: 'http://www.test.com' },
    { children: 'link', http: 'http://www.test.com' },
    { children: 'link', http: 'http://www.test.com' },
    { children: 'link', http: 'http://www.test.com' },
    { children: 'link', http: 'http://www.test.com' },
    { children: 'link', http: 'http://www.test.com' },
    { children: 'link', http: 'http://www.test.com' },
  ],
  logoLink: '',
  logoText: 'Logo',
  logoImgUrl: '#',
};
