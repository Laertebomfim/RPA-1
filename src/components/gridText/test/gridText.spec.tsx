import { render } from '@testing-library/react';
import GridText from '..';
import { mock } from '../mock';

describe('', () => {
  it('component is screen', () => {
    const { container } = render(
      <GridText TitleText="my title " html={mock.html} background={true}></GridText>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
