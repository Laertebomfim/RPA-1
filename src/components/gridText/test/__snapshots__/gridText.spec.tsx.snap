// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` component is screen 1`] = `
<section
  class="bg-white text-black"
  id="idSection"
>
  <div
    class="max-w-screenMD mx-auto min-h-screen relative"
  >
    <div
      class="text-center max-w-[580px] mx-auto pt-16 px-10"
    >
      <h2
        class="font-black font-montserrat text-black text-4xl md:text-6xl uppercase"
      >
        my title 
      </h2>
      <article
        class="[&_a]:text-red-500 [&_a]:hover:underline [&_a]:decoration-red-500 [&_ul]:py-10 [&_ol]:py-10 [&_p]:py-10 [&_p]:text-xl [&_h2]:text-2xl [&_h2]:font-bold [&_h2]:uppercase [&_h3]:text-2xl [&_h3]:font-bold [&_h3]:uppercase"
      >
        <p>
          hi 
        </p>
      </article>
      <article
        class="w-full  overflow-x-scroll"
      />
    </div>
  </div>
</section>
`;
